introspection-engine/connectors/sql-introspection-connector/src/calculate_datamodel.rs

```diff
@@ -137,6 +402,18 @@ fn calc_id_info(column: &Column, table: &Table) -> Option<IdInfo> {
     })
 }
 
+fn not_many_to_many_relation_name(fk: &ForeignKey, table: &Table) -> String {
+    let referenced_model = &fk.referenced_table;
+    let model_with_fk = &table.name;
+    let fk_column_name = fk.columns.get(0).unwrap();
+
+    if model_with_fk < referenced_model {
+        format!("{}_{}_{}", model_with_fk, fk_column_name, referenced_model)
+    } else {
+        format!("{}_{}_{}", referenced_model, model_with_fk, fk_column_name)
+    }
+}
```

Comment:
nitpick: it wasn't completely clear to me from the name what this function does, at first.